features/calculation_weekly_frequency.feature

```diff
@@ -23,12 +23,12 @@ Feature:
       And I should see my employer contributions for second period as "<employer_second_period>"
       And I should see my tax relief for second period as "<tax_relief_second_period>"
       And I should see my total contributions for second period as "<total_second_period>"
-      And I should see my employee contributions for third period as "<employee_third_period>"
+    And I should see my employee contributions for third period as "<employee_third_period>"
       And I should see my employer contributions for third period as "<employer_third_period>"
       And I should see my tax relief for third period as "<tax_relief_third_period>"
       And I should see my total contributions for third period as "<total_third_period>"
     Examples:
       | age | gender | salary | salary_frequency | contribution_preference | employee_percent | employer_percent | employee_current_period | tax_relief_current_period | employer_current_period | total_current_period | employee_second_period | employer_second_period | tax_relief_second_period | total_second_period | employee_third_period | employer_third_period | tax_relief_third_period | total_third_period |
-      | 24  | male   | 380    | per Week         | Minimum                 | 1                | 2                | £2.67                   | £0.53                     | £5.34                   | £8.01                | £8.01                  | £5.34                  | £1.60                    | £13.35              | £13.35                | £10.68                | £2.67                   | £24.03             |
-      | 32  | female | 400    | per Week         | Minimum                 | 3                | 2                | £8.61                   | £1.72                     | £5.74                   | £14.35               |                        |                        |                          |                     | £14.35                | £11.48                | £2.87                   | £25.83             |
-      | 24  | male   | 300    | per Week         | Full                    | 2                | 2                | £6.00                   | £1.20                     | £6.00                   | £12.00               | £9.00                  | £6.00                  | £1.80                    | £15.00              | £15.00                | £12.00                | £3.00                   | £27.00             |
+      | 24  | male   | 380    | per Week         | Minimum                 | 1                | 2                | £2.67                   | £0.53                     | £5.34                   | £8.01                | £8.01                  | £5.34                  | £1.60                    | £13.35              | £13.35                | £8.01                 | £2.67                   | £21.36             |
```

Comment:
yes, agreed. This was something I was starting to address yesterday. 
I will make an actual ticket for this and this will be something I'm going to continue investigating today to improve our cucumber features.
For now, I would like to get this merged as is and then have a PR with refactored cucumber, including this one. Just so we are showing the correct information ready for the demo